1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 82 Line 18_
```markdown

#### Pip  
* Pip - The default Python package manager, suitable for most Python packages.
    * Installation: `pip install package_name`

#### Mamba  
* Mamba - A faster alternative to Conda, using the same commands but significantly improving installation speed.
    * Installation: `mamba install package_name`
```
As of ~ this year, conda now uses mamba as it's default solver so we can drop this section.